development.rst

```diff
@@ -29,5 +29,9 @@ Making a release
 #. Create a signed tag: :code:`git tag -s v0.7.1`
 #. Push the commit and tag: :code:`git push origin refs/tags/v0.7.1`
 #. Create a release on GitHub
+  - For release notes:
+    - Main repo `git log $PREVIOUS_RELEASE_TAG...master --oneline --decorate >> commit_summary.txt`
```

Comment:
Maybe easier to maintain this snippet if it would be added as a script in the main repo?

Reply:
Hm yeah fair point. I'll do a PR for it in the main repo and then update this fork.

Reply:
Updated

Reply:
great! :)